### ISSUE TITLE
Seg fault on np.stack() of empty list

### ISSUE THREAD

**Issue by jeanlaroche** [serge-sans-paille/pythran#1451]
```
import numpy as np

#pythran export test(int)
def test(ii):
    mat=[]
    for u in range(ii):
        mat.append(np.zeros(10))
    return np.stack(mat,axis=0)
```
Try this with ii = 0. I get a segfault on the mac, latest version of pythran. To be fair, numpy issues a value error...

**Comment by serge-sans-paille** [comment]
Then we should issue a ValueError too. I think you can handle that, don't you? :-)

**Comment by jeanlaroche** [comment]
Yep. I'll get on it.

**Comment by serge-sans-paille** [comment]
See `pythran/pythonic/numpy/random/binomial.hpp` for an exemple.

**Comment by serge-sans-paille** [comment]
Fixed by #1453
